app/Utils/ScHearingType.cs

```diff
@@ -4,29 +4,43 @@ namespace SCJ.Booking.MVC.Utils
 {
     public static class ScHearingType
     {
-        //public const int AWS = 9104;
-        //public const int JMC = 9095;
-        //public const int PTC = 9543;
-        //public const int TCH = 9103;
+        public const int AWS = 9104;
+        public const int JMC = 9095;
+        public const int PTC = 9543;
+        public const int TCH = 9103;
 
         public const int TMC = 9090;
         public const int CPC = 9089;
         public const int JCC = 9005;
-        public const int TRIAL = 99999;
+        public const int TRIAL = 9001;
 
         public static string GetHearingType(int code)
         {
             return code == CPC ? nameof(CPC) : (code == JCC ? nameof(JCC) : nameof(TMC));
         }
 
-        public static readonly Dictionary<int, string> HearingTypeNameMap =
+        // map string ID to integer ID
+        public static readonly Dictionary <string, int> HearingTypeIdMap =
             new()
             {
-                //{AWS, "CV-Application Written Submissions (CV-AWS)"},
-                //{JMC, "Judicial Management Conference (JMC)"},
-                //{PTC, "CV-Pre-Trial Conference (CV-PTC)"},
-                //{TCH, "CV-Telephone Conference Hearing (CV-TCH)"},
+                { "AWS", AWS},
+                { "JMC", JMC},
+                { "PTC", PTC},
+                { "TCH", TCH},
+                { "Trials", TRIAL},
+                { "TMC", TMC},
+                { "CPC", CPC},
+                { "JCC", JCC},
+            };
 
+        // map integer ID to description text
+        public static readonly Dictionary<int, string> HearingTypeNameMap =
+            new()
+            {
+                { AWS, "CV-Application Written Submissions (CV-AWS)"},
+                { JMC, "Judicial Management Conference (JMC)"},
+                { PTC, "CV-Pre-Trial Conference (CV-PTC)"},
+                { TCH, "CV-Telephone Conference Hearing (CV-TCH)"},
```

Comment:
I re-enabled these types so they can be turned on via the API. We don't have emails for all of these types so it will throw an error right now, but that's true for the Trial type currently as well.

Reply:
It looks like we kept the email template in app/Views/ScBooking
- Email-CM-AWS.cshtml
- Email-CV-PTC.cshtml
- Email-CV-TCH.cshtml

Reply:
I saw those commented out, but didn't check if the templates were still there! In that case, I'll uncomment the block in ScBookingService to enable those emails (if they're ever used again)

Reply:
If they are ever used again, then I'm sure the templates will need to be rewritten anyway. My main goal of adding them back was to get rid of commented-out code that might accidentally get deleted rather than to make them fully functional.